### ISSUE TITLE
Support faker method options

### ISSUE THREAD

**Issue by antipopp** [kubb-project/kubb#886]
### What is the problem this feature would solve?

Currently the mapper in the faker js plugin will append the `()` to all the values, so that it cannot be customized.

```
mapper: {
          "status": "faker.random.arrayElement(['working', 'idle'])",
},
```

This example will eventually resolve to `faker.random.arrayElement(['working', 'idle'])()` which is wrong.

### What is the feature you are proposing to solve the problem?

Instead of appending the `()`, leave the decision to the user/config

**Comment by stijnvanhulle** [comment]
@antipopp I'm working on a refactor related to the mappers. When this is done, your issue here will be resolved.

**Comment by stijnvanhulle** [comment]
Fixed with v2.12.1
